src/CoreBundle/private/library/classes/TCMSTableEditor/TCMSTableEditorEndPoint.class.php

```diff
@@ -1700,6 +1701,9 @@ protected function _WriteDataToDatabase(&$oFields, &$oData, $bDataFromDatabase =
                     } else {
                         $query .= ', ';
                     }
+
+                    $postFieldsLanguageConvertedWithAccessRights[$sTargetFieldName] = $sqlValue;
```

Comment:
How does this contain "language" data? And how "access rights"?

Is this something like "all new db field values"?

Reply:
The wording was a bit off, we renamed it to make it clearer